module/VuFind/src/VuFind/RecordTab/OverdriveHoldingsFactory.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * Factory for building the Overdrive Holdings tab.
+ *
+ * PHP version 8
+ *
+ * Copyright (C) Villanova University 2019.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  RecordTabs
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @author   Brent Palmer <brent-palmer@icpl.org>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development Wiki
+ */
+
+namespace VuFind\RecordTab;
+
+/**
+ * OverdriveHoldingsFactory
+ *
+ * @category VuFind
+ * @package  RecordTabs
+ * @author   Brent Palmer <brent-palmer@icpl.org>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development Wiki
+ */
+class OverdriveHoldingsFactory implements \Laminas\ServiceManager\Factory\FactoryInterface
+{
+    /**
+     * Create an object
+     *
+     * @param ContainerInterface $container     Service manager
+     * @param string             $requestedName Service being created
+     * @param null|array         $options       Extra options (optional)
+     *
+     * @return object
+     *
+     * @throws ServiceNotFoundException if unable to resolve the service.
+     * @throws ServiceNotCreatedException if an exception is raised when
+     * creating a service.
+     * @throws ContainerException if any other error occurs
+     */
+    public function __invoke(\Interop\Container\ContainerInterface $container, $requestedName, array $options = null)
+    {
+        return new \VuFind\RecordTab\OverdriveHoldings();
```

Comment:
If you're just creating the object with no extra parameters, there is no need to build a whole factory. You can just configure the plugin manager to use `\Laminas\ServiceManager\Factory\InvokableFactory` to build the object. And speaking of that, I don't see any changes to register this new class in the PluginManager. I wonder if I accidentally lost those changes when resolving conflicts. Please let me know if you need help getting them back!

Reply:
Hmm. I don't think I ever put it in the PluginManager but it seems to be working without it listed. But I added it and removed the unnecessary factory and that works too.

Reply:
Thanks! The plugin managers have some default behavior for building objects when configuration is missing, so that's probably why this worked before... but I prefer to be explicit about configuration when possible, so thanks for adding the missing details!